TestServer/Unit/BookingServiceTests.cs

```diff
@@ -0,0 +1,110 @@
+using Moq;
+using server.Models.Domain;
+using server.Models.DTOs.Internal;
+using server.Models.DTOs.Request;
+using server.Repository;
+using TestServer.Unit;
+
+namespace server.Services.Internal.Tests
+{
+    public class BookingServiceTests : TestServiceBase<BookingService>
+    {
+        private readonly Mock<IBookingRepository> _bookingRepositoryMock;
+
+        public BookingServiceTests()
+        {
+            _bookingRepositoryMock = Mocker.GetMock<IBookingRepository>();
+        }
+        private UserClaims GetUserClaims()
+        {
+            return new UserClaims("name", "testUser", "noRole");
+        }
+
+        private UserClaims GetEventAdminClaims()
+        {
+            return new UserClaims("name", "testUser", "EventAdmin");
+        }
+
+
+        private CreateBookingRequest GetBookingRequest()
+        {
+            return new CreateBookingRequest { BookingDateTime = DateTime.Now, SeatId = 1 };
+        }
+
+        [Fact]
+        public async Task CreateBookingAsync_ValidBooking_ReturnsBookingDto()
+        {
+            // Arrange
+            var bookingRequest = GetBookingRequest();
+            var userClaims = GetUserClaims();
+
+            _bookingRepositoryMock.Setup(repo => repo.GetAsync()).ReturnsAsync(new List<Booking>());
+
+            // Act
+            var result = await Sut.CreateBookingAsync(bookingRequest, userClaims);
+
+            // Assert
+            Assert.NotNull(result);
+            Assert.Equal(userClaims.Objectidentifier, result.UserId);
+            Assert.Equal(userClaims.UserName, result.UserName);
+            Assert.Equal(bookingRequest.SeatId, result.SeatId);
+            Assert.Equal(bookingRequest.BookingDateTime.ToUniversalTime().ToString("o"), result.BookingDateTime);
+        }
+
+
+        [Fact]
+        public async Task CreateBookingAsync_SeatAlreadyBooked_ThrowsException()
+        {
+            // Arrange
+            var bookingRequest = GetBookingRequest();
+            var userClaims = GetUserClaims();
+            var bookingService = new BookingService(_bookingRepositoryMock.Object);
```

Comment:
den kan fjernes,  var bookingService = new BookingService(_bookingRepositoryMock.Object); bruke Sut isted for bookingService

Reply:
Åå, sorry. Jeg trodde du hadde lagt det til på alle testene på commiten din. Fikser nå : )